pages/common/hut.md

```diff
@@ -0,0 +1,29 @@
+# hut
+
+> A CLI tool for sr.ht.
+> Use `hg` in place of `git` below if using mercurial.
+> See `man hut` for more details.
```

Comment:
Typically we have a link to a webpage containing the man page of the program (manned.org) or (even better) to the program's documentation.

Reply:
[hut: A CLI tool for sr.ht](https://sr.ht/~emersion/hut/) is where you would link for that.  However it says  to read the man page to learn more.